syft/execution/placeholder.py

```diff
@@ -36,6 +36,7 @@ def __init__(
         self.child = None
         self.role = role
         self.tracing = tracing
+        self.inner = False
```

Comment:
If we keep this, could we find a more informative naming? Something like is_inner_state_placeholder?

Reply:
Good thought! Ended up getting rid of it though.